dev/io.openliberty.microprofile.telemetry.2.0.logging.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal/container/fat/TestUtils.java

```diff
@@ -103,4 +107,29 @@ public X509Certificate[] getAcceptedIssuers() {
             Log.error(TestUtils.class, "trustAll", e);
         }
     }
+
+    /*
+     * Ensure that the container was successfully started.
```

Comment:
It would be good to add comments on what this method does, and how long it waits and retries for.